GOVERNANCE.md

```diff
@@ -18,15 +18,65 @@ This project may include the following roles. Additional roles may be adopted an
+
+### 3.2. Decision-Making Guide
+
+This section describes how PyScript governing bodies (generally called “`maintainers`”) make decisions. This section provides clarity to all users, around the process around how decisions are made while providing a clear guide that can be consulted in case of doubts and the process to resolve conflicts and reach a resolution.
+
+We seek to honor the principles of collaboration, inclusive participation, and responsive decision making. Some aspects of this decision-making guide are required while others are provided as recommendations and are optional. We have clearly noted the optional aspects of decision making below.
+
+Finally, the PyScript `maintainters` group may either intervene, or be called upon, in the event that issues arise in the decision-making process and it comes short in providing the right guidance on specific subjects. This includes, but is not limited to: process ambiguity, violations of the decision-making process, mitigating circumstances that require process exceptions, etc.
+
+NOTE: The decision-making process is intended to balance broad participation of stakeholders with agility. This process aims to be a tool to help and not get in the way of the decision making process itself (turning it into a long and bureaucratic hassle), and is inspired by other successful projects such as the Python and Jupyter projects.
+
+### 3.2.1. Steps Towards Consensus
+
+**Informal consensus seeking.** Decision making starts with informal consensus seeking through discussion. The goal of this phase is to refine the proposal, consider alternatives, weigh trade-offs, and attempt to find informal consensus. The legitimacy of the consensus-seeking process is predicated on all stakeholders having their voices heard, so `maintainers` must be proactive in providing opportunities for all relevant stakeholders to provide input. If `maintainers` at informal consensus, they may immediately move to document and enact the decision. This is the consensus-seeking phase.
+
+**Calling a vote.** While "calling a vote" seems like a slow and bureaucratic process, its goal is to ensure that all stakeholders had a say in the process of making a decision. In an effort to keep things simple and lean, this step can be considered "integrated" into the previous (informal) step of Informal Consensus Seeking if there's enough support and agreement so that is clear that the proposal has the support from the majority of `maintainers`.
+
+If that's not the case and two or more parties are not aligned, any `maintainer` can call the matter to a vote. When that member (the sponsor) calls the vote, they shall summarize the proposal in its current form to make sure the proposal is clear to everyone. The summary and (and ensuing vote) shall be posted in a permanent public forum, with GitHub Discussions or Issues being the default preference.
```

Comment:
nit: should we say GitHub discussion is, in general, preferred over the forum? I am guilty myself of not being too active in the forum, but surely I'll get a notification if out of a GitHub discussion.

Reply:
Personally, I was not aware that the PyScript forum on the anaconda website existed before reading this.

Reply:
Yeah. It was active in the first 6 months (before we had GH Discussions) and the dynamics slowly migrated towards GH discussions and Discord. 

I'm +1 on removing that suggestion in favor of both.